README.md

```diff
@@ -208,6 +208,18 @@ arecord -f S16_LE -c1 -r 16000 -t raw -D default | nc localhost 43001
 
 - nc is netcat with server's host and port
 
+#### Client example on the Mac system:
+
+List of sounddevices:
+```
+ffmpeg -hide_banner -f avfoundation -list_devices true -i ""
+```
+This command will produce a list of audio devices with their corresponding IDs. Use the ID in the next command with the -i option:
```

Comment:
hey @SlavikCA maybe you can add something like this:

````suggestion
This command will produce a list of audio devices with their corresponding IDs. 
The output would look something like this:
```bash
...
[AVFoundation indev @ 0x123e05b20] AVFoundation audio devices:
[AVFoundation indev @ 0x123e05b20] [0] MacBook Air Microphone
[AVFoundation indev @ 0x123e05b20] [1] Device 1
[AVFoundation indev @ 0x123e05b20] [2] Device 2
...
```
Use the ID in the next command with the -i option. In this example I'm using a `MacBook Air` and the microphone is the device number `0`. 
````